### PR TITLE
Get volume request should show only list of volume name and respectiv…

### PR DESCRIPTION
**Pull request by gaurav36**
…e uuid

Currently, when you make a volume Get request then it will display list
of whole volume info (volinfo) object.

With this patch it will  show list of volume name and associated volume uuid
when user make get volume request.

Issue #44

Signed-off-by: Gaurav Kumar Garg <garg.gaurav52@gmail.com>

### CODE REVIEW

File: volume/store-utils.go [hunk 1]
```diff
@@ -56,13 +56,13 @@ func DeleteVolume(name string) error {
 
 //GetVolumes retrives the json objects from the store and converts them into
 //respective volinfo objects
-func GetVolumes() ([]Volinfo, error) {
+func GetVolumes() ([]Volume_list, error) {
```
Comment (atinmu) [inline volume/store-utils.go]: Almost all go projects follow lower camel cases. Could you change the variable name to something like volList?

**Comment by atinmu** [comment]
Please ensure you compile the code with 'make install' before sending the PR. Looking at the CI failure its straight forward that the changes break compilation.

File: volume/struct.go [hunk 2]
```diff
@@ -50,6 +50,13 @@ const (
 	DistDispStripe
 )
 
+// Volume list
+// Represent the list of volume during GetVol request
+type Volume_list struct {
```
Comment (kshlm) [inline volume/struct.go]: A simple map[string]string should suffice for this. No need to create a whole new type.

**Comment by kshlm** [comment]
Please run `make verify` and `make build` before sending PRs.

File: volume/store-utils.go (hunk 1 above)
Comment (gaurav36) [inline volume/store-utils.go]: solved in latest patch set.

File: volume/struct.go [hunk 3]
```diff
@@ -50,6 +50,13 @@ const (
 	DistDispStripe
 )
 
+// Volume list
+// Represent the list of volume during GetVol request
+type VolumeList struct {
```
Comment (kshlm) [inline volume/struct.go]: Like I mentioned in my previous comment, there is no need for a separate type for this. A map is good enough.

**Comment by atinmu** [comment]
@gaurav36 github model works with every changes as separate commits. I think you have followed a gerrit workflow where you have rebased your changes on the same commit which is unacceptable in this model. Now onwards ensure to send separate commits.

**Pull request merged**